build/cqs.tsv

```diff
@@ -28,24 +28,16 @@ com.googlecode.juniversalchardet:juniversalchardet	1.0.3	compile
 com.google.errorprone:error_prone_annotations	2.3.3	compile
 com.google.flatbuffers:flatbuffers-java	1.9.0	compile
 com.google.guava:guava	11.0.2	compile
-com.google.guava:guava	12.0.1	compile
-com.google.guava:guava	14.0.1	compile
-com.google.guava:guava	17.0	compile
-com.google.guava:guava	18.0	compile
+com.google.guava:guava	27.0-jre	compile
```

Comment:
Already approved here: https://dev.eclipse.org/ipzilla/show_bug.cgi?id=21084.  

@elahrvivaz just to confirm, your understanding is that we needn't enter piggyback CQs anymore, right?

Reply:
that's my understanding based on the cq page verbiage, but maybe we should confirm with Wayne or someone at eclipse.